### PR TITLE
Cu 2wg9u13 vesting schedule

### PR DESCRIPTION
**Pull request by easteregg**
## Issue
https://app.clickup.com/t/2wg9u13

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2wg9u13 Implement vesting schedule ids on picasso bonds](https://app.clickup.com/t/2wg9u13)

File: frontend/apps/picasso/components/Atom/TokenAsset.tsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ export const TokenAsset: React.FC<TokenAssetProps> = ({
   label,
   ...rest
 }) => {
-  const token = getToken(tokenId);
+  const token = getToken(tokenId.toLowerCase());
+  if (!token) console.log(tokenId);
```
Comment (andresmechali) [inline frontend/apps/picasso/components/Atom/TokenAsset.tsx:18]: Should the `console.log` be removed?

File: frontend/apps/picasso/apollo/queries/vestingSchedule.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { gql } from "@apollo/client";
+
+export const GET_VESTING_SCHEDULE_BY_ADDRESS = gql`
```
Comment (andresmechali) [inline frontend/apps/picasso/apollo/queries/vestingSchedule.ts:3]: Do you need all the data here?

File: frontend/apps/picasso/components/Organisms/Bond/BondForm.tsx [hunk 3]
```diff
@@ -182,6 +182,10 @@ export const BondForm: FC<{
     return <PositionDetails>{children}</PositionDetails>;
   };
 
+  console.log({
```
Comment (andresmechali) [inline frontend/apps/picasso/components/Organisms/Bond/BondForm.tsx:185]: Same here

**Review by andresmechali — APPROVED** [review]
LGTM

**Review by saadjhk — APPROVED** [review]

File: frontend/apps/picasso/apollo/queries/vestingSchedule.ts (hunk 2 above)
Comment (easteregg) [reply frontend/apps/picasso/apollo/queries/vestingSchedule.ts:3]: removed.

**Comment by easteregg** [comment]
@Mergify refresh

**Pull request merged**